plugins/koa/index.js

```diff
@@ -0,0 +1,13 @@
+
+module.exports = serverPlugin;
+
+function serverPlugin() {
```

Comment:
Let's keep the name consistent with `const koaServer = require('@reframe/koa');`